news/_posts/2021-10-01-hacktoberfest-2021.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "Hacktoberfest 2021"
+subtitle: "October is here, so is Hacktoberfest!"
+date:  2021-10-01 09:00:00
+authors: [kpodemski]
+image: /assets/images/2021/10/hacktoberfest.svg
+icon: icon-github-alt
+tags: [community, contribution, github, contribute]
+---
+
+Hacktoberfest is a month-long celebration of open-source software that happens online every October, sponsored by Digital Ocean, the American cloud hosting company [DigitalOcean](https://www.digitalocean.com). People are encouraged to engage and create at least 4 pull requests for open-source projects on GitHub.
+
+If you want to contribute to open-source projects but don’t know where to start, Hacktoberfest might be the perfect opportunity for you.
+
+
+![Hacktoberfest 2021](/assets/images/2021/10/hacktoberfest.svg)
```

Comment:
Should we add a mention to say this image does not belong to us? Or does Hacktoberfest clearly mention we can reuse without quoting author?

Reply:
I got it from there:
https://hacktoberfest.digitalocean.com/brandguidelines

We can use it to promote event